### PR TITLE
Integrated stale pull requests API

### LINKED ISSUE #28: Integrate stale pull requests API
We are now having `https://staging-api.realdevsquad.com/pullrequests/stale` route, which shows newest open PRs - easy to track which are the PRs which are open from a long time, which have NOT been merged yet. So they can be taken care of.
Now we need to have a page on the status site to make use of this route and show the newest open PRs.

### PR DESCRIPTION
**Pull request by adnanraghib1308**
Closes #28 
Integrated stale pull requests API. This is solution for the issue #28 . See Screenshot

Screenshot 2021-01-08 at 10 13 32 AM

### CODE REVIEW

File: src/components/pullRequests/card/card.module.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.card {
+  border: 0.2em solid black;
+  border-radius: 0.5em;
+  padding: 1em;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  max-width: 850px;
+  margin-bottom: 2em;
```
Comment (whyDontI) [inline src/components/pullRequests/card/card.module.scss:9]: Please use px as a unit for border, margin, padding, these metrics are not dependent on the parents/roots font-size.
https://css-tricks.com/rems-ems/

File: src/components/pullRequests/card/card.module.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+.activeBtn{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #4fb90d;
+  color: white;
+  border: 2px solid black;
+  border-radius: 5px;
+  padding: 0.5rem 2rem;
```
Comment (whyDontI) [inline src/components/pullRequests/card/card.module.scss:52]: Please use px as a unit for border, margin, padding, these metrics are not dependent on the parents/roots font-size.
https://css-tricks.com/rems-ems/

File: src/components/pullRequests/card/card.module.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  font: inherit;
+  cursor: pointer;
+  font-weight: bolder;
+  font-size: 20px;
+
```
Comment (whyDontI) [inline src/components/pullRequests/card/card.module.scss:57]: Can we remove blank lines?

File: src/components/pullRequests/section/index.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+type Props = {
+  heading: string,
+  content: []
+}
+
+const Section: FunctionComponent<Props> = ({ heading, content }) => {
+
+  const cards = content.map((pullRequest, i) => {
+    return <Card pullRequest={pullRequest} key={i} />
+  })
```
Comment (whyDontI) [inline src/components/pullRequests/section/index.tsx:14]: Cards will re-render and reinitialized every time `Section` updates
Can we make it a separate function and pull it out of the `Section` function definition?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Requested some little changes 👍🏽

**Comment by adnanraghib1308** [comment]
Made the required changes. Please review again. @whyDontI

**Review by whyDontI — CHANGES_REQUESTED** [review]
As per your current design. All the cards are stacked vertically which leaves a lot of space vacant in Desktop view.
Vertical cards are good for devices with small screens, but we should use the space in Desktop mode.

Can you make the cards display horizontally in 2 or columns, similar to what we have done in the `Tasks` tab?

Thanks
Let me know if you need any help 😄 👍🏽

File: src/components/pullRequests/card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+          {informationElement('Updated At', fromNowUpdatedAt)}
+        </span>
+        <span className={classNames.onhover}>
+          {informationElement('Updated At', readableUpdatedAt)}
+        </span>
+      </div>
+      {informationElement('PR url', url)}
+      {informationElement('Username', username)}
+      <div className={classNames.Center}>
+        <a href = {url} className={classNames.links} target = "_blank">
```
Comment (sumitd94) [inline src/components/pullRequests/card/index.tsx:66]: Please make sure you add a `noreferrer` attribute for `href` tags.

**Review by sumitd94 — CHANGES_REQUESTED** [review]

**Comment by adnanraghib1308** [comment]
@whyDontI @sumitd94 Done with changes. Please review again

File: src/components/pullRequests/card/card.module.scss [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  border: 2px solid black;
+  border-radius: 10px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin-bottom: 30px;
+  margin-left: 60px;
+  margin-right: 60px;
+}
```
Comment (sumitd94) [inline src/components/pullRequests/card/card.module.scss:11]: NIT : Here you can use the shorthand property like `margin: 0 60px 30px 60px;`

File: src/components/pullRequests/card/card.module.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+.prTitle {
+  color: #041484;
+  font-size: 25px;
+  font-weight: bold;
+}
+
+.cardFooter {
+  display: flex;
+  flex-direction: row;
```
Comment (sumitd94) [inline src/components/pullRequests/card/card.module.scss:21]: NIT : By default when you use `display: flex` the direction is `row` . so `flex-direction: row` is not necessary.

File: src/components/pullRequests/card/card.module.scss [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+  background: #4fb90d;
+  color: white;
+  border: 2px solid black;
+  border-radius: 5px;
+  padding: 8px 25px;
+  font: inherit;
+  cursor: pointer;
+  font-weight: bolder;
+  font-size: 20px;
+}
```
Comment (sumitd94) [inline src/components/pullRequests/card/card.module.scss:46]: NIT: you can even add a `outline: none` property.
Take a look here what this is useful for.https://developer.mozilla.org/en-US/docs/Web/CSS/outline

**Review by sumitd94 — DISMISSED** [review]

**Review by whyDontI — APPROVED** [review]
Good work @adnan1308 👍🏽

**Pull request merged**